packages/tests/src.ts/test-contract.spec.ts

```diff
@@ -336,6 +338,72 @@ describe("Test Contract Transaction Population", function() {
         });
         assert.strictEqual(BigNumber.from(balance).toNumber(), 10000, 'balance mismatch');
     }).timeout(60000);
+
+    it("should be able to call contract methods", async function () {
+        // configs
+        const providerTestnet = ethers.providers.getDefaultProvider('testnet');
+        const contractHederaEoa = {
+            "account": '0.0.29562194',
+            "privateKey": '0x3b6cd41ded6986add931390d5d3efa0bb2b311a8415cfe66716cac0234de035d'
+        };
+        const clientHederaEoa = {
+            "account": "0.0.28542425",
+            "privateKey": "302e020100300506032b65700422042077d69b53642df4e59215da8f5f10c97f6a6214b6c8de46940d394da21d30e7cc"
+        };
+
+        // contract init
+        // @ts-ignore
```

Comment:
Why do we need `ts-ignore`?

Reply:
That's the exception:
```
TS2345: Argument of type '{ account: string; privateKey: string; }' is not assignable to parameter of type 'BytesLike | SigningKey | ExternallyOwnedAccount'.
```
And the code piece:
```
const contractHederaEoa = {
    "account": '0.0.x',
    "privateKey": '0x0'
};
const contractWallet = new ethers.Wallet(contractHederaEoa, providerTestnet);
```
Maybe we should wrap the credentials in an object before we pass them to the Wallet? Is it another task or what should I do in that scenario?